nss-ivory-trolls/BangazonAPI
# Order and Payment type controllers

# Description

Added order and payment type 

## Type of change



# Testing Instructions
git fetch --all
git checkout AP-Controller

Check CRUD for Order and PaymentType to ensure those controllers are working


# Checklist:
